crates/rolldown/tests/rolldown/function/platform/node/should_not_throw_warnings_for_import_builtin_modules/basic/artifacts.snap

```diff
@@ -9,8 +10,9 @@ source: crates/rolldown_testing/src/integration_test.rs
 import * as fs from "fs";
 import * as nodeFs from "node:fs";
 
+
 //#region main.js
-console.log(fs, nodeFs, require("path"), require("node:path"));
+console.log(fs, nodeFs, __require("path"), __require("node:path"));
```

Comment:
`esbuild` will auto convert `format` to `cjs` if you don't specify `format` when `platform` is `node`